packages/module-loader-commonjs/src/__tests__/index.test.ts

```diff
@@ -52,4 +59,30 @@ describe('createCommonJsModuleLoader', () => {
 
     expect(loadedModule).toEqual(expectedModule);
   });
+
+  it('sets agent if process.env.HTTP_PROXY is defined', async () => {
+    const currentHttpProxy = process.env.HTTP_PROXY;
+    const url = 'http://example.com/test.js';
+
+    process.env.HTTP_PROXY = 'http://';
```

Comment:
We have an undocumented newline rule in this project. Please add a newline between line 67 and 68 :)

Reply:
Hi @clebert I fixed it :)